cmd/jujud/agent/machine/manifolds.go

```diff
@@ -926,6 +928,22 @@ func IAASManifolds(config ManifoldsConfig) dependency.Manifolds {
 			FanConfigurerName: fanConfigurerName,
 		})),
 
+		// DBAccessor is a manifold that provides a DBAccessor worker
+		// that can be used to access the database.
+		dbAccessorName: ifController(dbaccessor.Manifold(dbaccessor.ManifoldConfig{
+			AgentName:            agentName,
+			QueryLoggerName:      queryLoggerName,
+			Clock:                config.Clock,
+			Hub:                  config.CentralHub,
+			Logger:               loggo.GetLogger("juju.worker.dbaccessor"),
+			LogDir:               agentConfig.LogDir(),
+			PrometheusRegisterer: config.PrometheusRegisterer,
+			NewApp:               dbaccessor.NewApp,
+			NewDBWorker:          dbaccessor.NewTrackedDBWorker,
+			NewMetricsCollector:  dbaccessor.NewMetricsCollector,
+			NewNodeManager:       dbaccessor.IAASNodeManager,
+		})),
```

Comment:
@jack-w-shaw you need to copy this to `CAASManifolds` and ensure the has the following: `NodeNodeManager: dbaccessor.CAASNodeManager`

Reply:
Done